indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -92,14 +92,15 @@ def _check_access(self):
         if not self._user_is_authorized_submitter():
             # XXX: should event managers be able to submit on behalf of the user?
             raise Forbidden
-        # TODO: check if submitting papers for editing is allowed in the event
```

Comment:
I think it's be worth keeping the TODO comment since we'll still need to check whether submission is currently open for the given editable type.